### PR TITLE
Updated tables

### PR DESCRIPTION
**Pull request by yashodgayashan**

### CODE REVIEW

File: BackendNew/scripts/create-tables.sql [hunk 1]
```diff
@@ -3,27 +3,26 @@
...
 CREATE TABLE Role (
-    RoleId int NOT NULL AUTO_INCREMENT,
     Name varchar(255) NOT NULL,
     Description varchar(255),
-    PRIMARY KEY (RoleId)
+    PRIMARY KEY (Name)
 );
 
 CREATE TABLE Admin (
     AdminId int NOT NULL AUTO_INCREMENT,
```
Comment (hasinichathu) [inline BackendNew/scripts/create-tables.sql:12]: Can we have auto incremented number as username?

File: BackendNew/scripts/create-tables.sql [hunk 2]
```diff
@@ -3,27 +3,26 @@
...
-    RoleId int,
+    RoleName int,
```
Comment (hasinichathu) [inline BackendNew/scripts/create-tables.sql:13]: Role Name should be VARCHAR. Right?

File: BackendNew/scripts/create-tables.sql [hunk 3]
```diff
@@ -3,27 +3,26 @@
...
     Name varchar(255) NOT NULL,
     PhoneNumber int,
     Email varchar(255),
```
Comment (hasinichathu) [inline BackendNew/scripts/create-tables.sql:16]: Can you Email NOT NULL

File: BackendNew/scripts/create-tables.sql [hunk 4]
```diff
@@ -3,27 +3,26 @@
...
     PRIMARY KEY (AdminId),
-    FOREIGN KEY (RoleId) REFERENCES Role(RoleId)
+    FOREIGN KEY (RoleName) REFERENCES Role(Name)
 );
 
 CREATE TABLE Company (
     CompanyId int NOT NULL AUTO_INCREMENT,
     CreatedBy int NOT NULL,
-    RoleId int NOT NULL,
+    RoleName int NOT NULL,
```
Comment (hasinichathu) [inline BackendNew/scripts/create-tables.sql:25]: RoleName should be VARCHAR.

File: BackendNew/scripts/create-tables.sql [hunk 5]
```diff
@@ -61,10 +60,9 @@ CREATE TABLE Vacancy (
 );
 
 CREATE TABLE Student (
-    StudentId int NOT NULL AUTO_INCREMENT,
     BatchId int NOT NULL,
     CreatedBy int NOT NULL,
-    RoleId int NOT NULL,
+    RoleName int NOT NULL,
```
Comment (hasinichathu) [inline BackendNew/scripts/create-tables.sql:65]: RoleName should be VARCHAR

File: BackendNew/scripts/create-tables.sql [hunk 6]
```diff
@@ -82,32 +80,37 @@ CREATE TABLE Student (
     DateOfStart DATE,
     Degree varchar(100),
     Password varchar(100),
```
Comment (hasinichathu) [inline BackendNew/scripts/create-tables.sql:82]: I have a doubt.  Suppose a student registered for a internship and could not complete it properly. Then he again register for internship. According to this "STUDENT" table he should have 2 accounts with same username(student ID) with different passwords.

File: BackendNew/scripts/create-tables.sql (hunk 1 above)
Comment (yashodgayashan) [reply BackendNew/scripts/create-tables.sql]: No I make it as a Name in the latest version

File: BackendNew/scripts/create-tables.sql (hunk 2 above)
Comment (yashodgayashan) [reply BackendNew/scripts/create-tables.sql]: Yes.

File: BackendNew/scripts/create-tables.sql (hunk 3 above)
Comment (yashodgayashan) [reply BackendNew/scripts/create-tables.sql:16]: It depends. But the admin id should be filled

File: BackendNew/scripts/create-tables.sql (hunk 4 above)
Comment (yashodgayashan) [reply BackendNew/scripts/create-tables.sql]: Added

**Comment by yashodgayashan** [comment]
@hasinichathu I have updated the PR could you please check and review. Thank you.

File: BackendNew/scripts/create-tables.sql (hunk 3 above)
Comment (hasinichathu) [reply BackendNew/scripts/create-tables.sql:16]: So in case if password is forgotten

Comment (yashodgayashan) [reply BackendNew/scripts/create-tables.sql:16]: Okay I will make it NOT NULL

**Comment by yashodgayashan** [comment]
@hasinichathu  resolved requested changes

**Pull request merged**